githooks/README.md

```diff
@@ -0,0 +1,30 @@
+# Git hook for local pre-commit linting
```

Comment:
Can this be executed on Windows?

Reply:
It should, with the correct installation of `markdownlint` and `git-clang-format`.

Reply:
Perfect!